tools/halide_benchmark.h

```diff
@@ -1,40 +1,16 @@
 #ifndef BENCHMARK_H
 #define BENCHMARK_H
 
+#include <chrono>
 #include <limits>
 
+namespace Halide {
+namespace Tools {
+
 // Benchmark the operation 'op'. The number of iterations refers to
 // how many times the operation is run for each time measurement, the
 // result is the minimum over a number of samples runs. The result is the
 // amount of time in seconds for one iteration.
-#ifdef _WIN32
```

Comment:
I built&ran test/performance on my local Windows box and numbers look comparable before and after. Additionally, various notes from MSVC team indicate that VC2015 now has a good implementation for it. So for our purposes I think we're fine.